packages/js/CHANGELOG.md

```diff
@@ -3,8 +3,11 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.1.1]
+### Fixed
```

Comment:
Can you add this under the [Unreleased] block and add a date (like the other tags).